### PR TITLE
styling fav buttons and navbar icons

### LINKED ISSUE #24: Likes button and show number of likes
Likes button is not independent from review to review yet :(

### PR DESCRIPTION
**Pull request by dknv**
put styles for fav buttons and navbar icons

### CODE REVIEW

File: client/src/components/Navbar.js [hunk 1]
```diff
@@ -22,10 +21,10 @@ function Navbar() {
             <Link to="/"> <h1 className={styles.name}>bookworms</h1> </Link>
             <div className={styles.links}>
                 <Tooltip title="Add Reviews">
-                    <Link to ="/create"><AddIcon className={styles.button} fontSize="large"/></Link>
+                    <Link to ="/create"><Icon color="primary" className={styles.button} fontSize="large">add</Icon></Link>
                 </Tooltip>
                 <Tooltip title="Sign In">
-                    <SigninIcon className={styles.button} fontSize="large" onClick={() => setSignInShow(true)}/>
+                    <Icon color="primary" className={styles.button} fontSize="large" onClick={() => setSignInShow(true)}>local_library</Icon>
```
Comment (dknv) [inline client/src/components/Navbar.js:27]: I can't seem to change the color of icons imported from`material-ui/icons/` so I switched to `material-ui/core/Icon`... if it is not preferable, please let me know and I'll try to work with it again!

File: client/src/components/ReviewList.js [hunk 2]
```diff
@@ -1,16 +1,32 @@
...
+    //         body: JSON.stringify({review_id: reviewId})
+    //     })
+    //     // if the request wasn't successful, throw an error for the user to know 
+    //     if (!response.ok) {
+    //         const message = `An error has occured: ${response.status}`;
+    //         throw new Error(message);
+    //     } else {
+    //         console.log("fav updated");
+    //     }
+    // }
```
Comment (dknv) [inline client/src/components/ReviewList.js:28]: this is for integrating with likes API -- will get back on this soon

File: client/src/components/ReviewList.js [hunk 3]
```diff
@@ -21,9 +37,9 @@ function ReviewList ({reviews}) {
                     <p> {r.text} </p>
                     <div className={styles.footer}>
                         <p> {r.categories.join(', ')} </p>
-                        <FavIcon 
-                            className={styles.footerRight} 
-                            onClick={toggleFavorite}/>
+                        <FavIcon/>
+                            {/* onClick={toggleFav(r.id)}/> */}
```
Comment (dknv) [inline client/src/components/ReviewList.js:41]: commented out as it is related to likes API

File: client/src/components/Navbar.js (hunk 1 above)
Comment (jeromewu) [reply client/src/components/Navbar.js:27]: I think it is OK for now.

**Review by jeromewu — APPROVED** [review]
It is good to have PR with small changes. :smile:

**Comment by jeromewu** [comment]
Merged, thanks for the hard work. :+1:

**Pull request merged**